src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1242,8 +1248,31 @@ def remove_temp_test_dir(self):
             errors.append("Error removing temporary test files")
         return errors
 
+    def fail(self, msg=None):
+        if self.dumped_engines_stacks is False:
+            self.dumped_engines_stacks = True
+            self.log.info("Test status has failed, dumping ULT stacks")
+            dump_engines_stacks(self.hostlist_servers)
+        super().fail(msg)
+
+    def error(self, msg=None):
```

Comment:
(pylint-arguments-renamed) Parameter 'message' has been renamed to 'msg' in overridden 'TestWithServers.error' method